ui/admin/app/styles/app.scss

```diff
@@ -976,3 +976,13 @@
     }
   }
 }
+
+// form section
+.form-section {
```

Comment:
thoughts on calling this something else to not confuse it with the actual form? 
thinking out loud - maybe `ldap-section`?

Reply:
updated!